unlock-protocol.com/blog/forbes-ads-free.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Unlocking an ads-free Forbes
+subTitle: Memberships can have diverse benefits!
+authorName: Julien Genestoux
+publishDate: December 17, 2019
+description: Forbes introduced their very first membership today and it lets anyone with an ethereum wallet unlock an ads free experience
+image: /static/images/blog/forbes-ads-free/forbes.svg
+---
+
+At Unlock, we believe that the attention driven economy will be replaced by a membership driven economy. In today's web, creation is rewarded if it is able to attract the eyes, rather than expand the sight: this brought us information overload, endless slideshows, click-bait and fake news.
```

Comment:
I am not sure if the meaning behind `rather than expand the sight` is clear

Reply:
I read it as meaning something like broadening horizons

Reply:
```suggestion
At Unlock, we believe that the attention driven economy will be replaced by a membership driven economy. In today's web, creation is rewarded if it is able to attract the eyes, rather than expand the sight: this brought us information overload, endless slideshows, clickbait and fake news.
```

Also I think "clickbait" is the more common usage over "click-bait"